src/Servers/Kestrel/Transport.Quic/src/QuicTransportOptions.cs

```diff
@@ -25,22 +25,22 @@ public class QuicTransportOptions
         /// <summary>
         /// The Application Layer Protocol Negotiation string.
         /// </summary>
-        public string Alpn { get; set; }
+        public string? Alpn { get; set; }
```

Comment:
I'm guessing that if we ship this it won't be nullable. But given there's no default value at the moment, I think this is fine for now.